src/system-core.js

```diff
@@ -74,6 +74,8 @@ systemJSPrototype.getRegister = function () {
   return _lastRegister;
 };
 
+systemJSPrototype.errMsg = errMsg;
```

Comment:
I take it you are using this for further customization hooks? Is it really strictly necessary?

Reply:
This is necessary for use in the extras. We could rename it to be shorter, but it has to be exposed so that amd.min.js and module-types.min.js can benefit from the byte savings.

Reply:
I would actually be ok with duplication of the code in those extras through importing the common function and in turn bundling it again. 20 bytes against SystemJS is not an issue. But s.js must stay small, especially into the long term future of the project.

The SystemJS build should naturally dedupe it anyway though.

Reply:
Ah I didn't realize - we can now import things into our extras, since they're going through rollup. I'll update them.

Also, we could remove the iife's in the extras' src code, since that's now being handled by rollup/terser